### DISCUSSION TITLE
Correct pronunciation of Statiq

### DISCUSSION THREAD

**Discussion by Swimburger** [statiqdev/Discussions#93] [Q&A] [answered]
What is the correct pronunciation of Statiq?
I heard some folks pronounce it like the word "Static", but my original guess was that the end of "Statiq" is pronounced as "Un**ique**".

**Comment by devlead** [comment, marked as answer]
Here you can here the author talk about it
https://youtu.be/43oQTRZqK9g

**Reply by daveaglick** [reply]
The only problem is that the author isn't exactly consistent either 😂

I'll make a ruling in my capacity at benevolent dictator here and now that it's pronounced like "static".

![](https://media.giphy.com/media/3orieOPmJGswsJ7Ily/giphy.gif)

**Reply by Swimburger** [reply]
Thanks, "static" it is! I'll accept this as answer, once GitHub allows me to press the button.

**Reply by Swimburger** [reply]
I can't accept the answer, only people with "triage access" can according to GH support. If anyone else can accept, please do.

**Reply by daveaglick** [reply]
Interesting! I would have guessed the poster could mark an answer, but I guess the intent is to make sure answers are verified...or something.
